frontend/src/scenes/insights/ActionFilter/ActionFilterRow/index.tsx

```diff
@@ -109,53 +111,60 @@ export function ActionFilterRow({
         value = entity.id || filter.id
     }
 
+    const orLabel = showOr ? <div className="stateful-badge mc-main or width-locked">OR</div> : null
+
     return (
         <div className={horizontalUI ? 'action-row-striped' : ''}>
-            {showOr && (
-                <Row align="middle">
-                    {index > 0 && (
-                        <div className="stateful-badge mc-main or width-locked" style={{ marginTop: 12 }}>
-                            OR
-                        </div>
-                    )}
+            {!horizontalUI && index > 0 && (
+                <Row align="middle" style={{ marginTop: 12 }}>
```

Comment:
Is there a reason why we'd want this empty row to render if `orLabel` is null? (Spacing perhaps?)

Reply:
Nope, I was actually abstracting duplicated checks into the `orLabel` constant, but this leads to an incorrect render, good catch!